src/data/list/basic.lean

```diff
@@ -552,8 +552,11 @@ by rw [concat_eq_append, reverse_append, reverse_singleton, singleton_append]
 @[simp] theorem reverse_reverse (l : list α) : reverse (reverse l) = l :=
 by induction l; [refl, simp only [*, reverse_cons, reverse_append]]; refl
 
+theorem reverse_involutive : involutive (@reverse α) :=
```

Comment:
```suggestion
@[simp] theorem reverse_involutive : involutive (@reverse α) :=
```

Reply:
Should _every_ involutive statement in mathlib be tagged `simp`?